.clang-format

```diff
@@ -17,3 +17,24 @@ BraceWrapping:
   BeforeCatch: true
   BeforeElse: true
 
+IncludeBlocks: Regroup
+IncludeCategories:
+- Regex:           '^"power_grid_model\.hpp"$'
```

Comment:
why not just use the one below? i'm not sure whether this should be a separate `Priority` (group). Instead, you could make it `Priority: 1` and `SortPriority: 0`, but personally, I believe you can just leave this entry out.